pom.xml

```diff
@@ -31,7 +31,7 @@
         <spring-cloud.version>2022.0.4</spring-cloud.version>
         <chaos-monkey-spring-boot.version>2.3.10</chaos-monkey-spring-boot.version>
         <jolokia-core.version>1.7.1</jolokia-core.version>
-
+        <lombok.version>1.18.30</lombok.version>
```

Comment:
Thanks for your contribution
Could you please add a comment line to keep a trace of this hack. We should remove it while a future upgrade.
Thanks a lot